docs/faq.md

```diff
@@ -0,0 +1,89 @@
+
+### Q: Who maintains Headlamp?
+
+**A:** Headlamp's core team is composed of developers that are employed by Microsoft. As a 100% open source project and a CNCF Sandbox project, Headlamp has a transparent governance and it encourages any users/developers to participate in it.
+
+### Q: What capabilities / credentials does Headlamp require to access my Kubernetes cluster?
+
+**A:** Headlamp doesn't require access to the cluster in itself. It instead relies to RBAC when connecting to the Kubernetes API server, meaning that it's the users that are required the credentials to access the cluster (via a service token or client certificate). Headlamp may store the token in the browser's local storage, but never in its backend/server.
+
+### Q: Is Headlamp customizable?
+
+**A:** Yes, Headlamp is highly customizable, thanks to its robust plugin system. This system allows for the extension of Headlamp's core functionalities, catering to specific use cases and workflows. For more information on creating and managing plugins, visit our [plugins page](https://www.headlamp.dev/docs/latest/plugins/).
+
+### Q: How often is Headlamp updated?
+
+**A:** Headlamp tries to have a new features version released every month. Sometimes, there may be delays for a couple of weeks. Bug fix versions can be released in between feature versions, whenever appropriate (releases fixing critical bugs are often released quickly after a fix is added).
+
+---
+
+## Installation and Setup
+
+### Q: How can I install and access Headlamp?
+
+**A:** To install Headlamp, follow the detailed instructions provided in the [official installation guide](https://www.headlamp.dev/docs/latest/installation/). The process will guide you through downloading the application, setting up your Kubernetes cluster access, and launching Headlamp to connect to your cluster. For desktop applications, you can find additional information in the [desktop installation guide](https://www.headlamp.dev/docs/latest/installation/desktop/).
+
+### Q: Can I install Headlamp in my Kubernetes cluster?
+
+**A:** Absolutely, Headlamp can be deployed directly within your Kubernetes cluster. Detailed instructions for in-cluster installation can be found in the [in-cluster installation guide](https://www.headlamp.dev/docs/latest/installation/in-cluster/).
+
+---
+
+## Usage and Features
+
+### Q: Can I monitor multiple clusters with Headlamp?
+
+**A:** Yes, Headlamp allows you to monitor multiple clusters. You can switch between different clusters using the cluster switcher in the Headlamp interface.
+
+### Q: Can I manage my Kubernetes resources directly through Headlamp?
+
+**A:** Yes, Headlamp enables direct management of Kubernetes resources through its user interface. It reflects the user's role and permissions, for instance, if a user does not have deletion permissions, the delete button will not be displayed. This ensures a secure and role-compliant interaction with cluster resources.

```

Comment:
Maybe "....its user interface as allowed by the user's role and permissions. For instance, ...". I don't think we need the last sentence.